postgis/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM timescale/timescaledb:0.0.11-beta
+FROM timescale/timescaledb:latest
```

Comment:
Maybe not, but I also worry that it will be forgotten when updating versions and inadvertently left behind. If you prefer a specific version, I can revert.